src/sgf_parser/models/method_dt.py

```diff
@@ -0,0 +1,49 @@
+from decimal import Decimal
+from typing import Literal
+
+from pydantic import Field, computed_field
+
+from sgf_parser.models import MethodType, MethodData, Method
+
+
+class MethodDTData(MethodData):
+    """
+    Method Pore Pressure Dissipation Test (placeholder)
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    elapsed_time: Decimal | None = Field(None, alias="AD", description="Elapsed time (s)")
```

Comment:
Suggest renaming `elapsed_time` to just `time` so it is equal to the attribute used in Simple Sounding (SS).